python-client/api-reference/step.mdx

```diff
@@ -9,18 +9,29 @@ Checkout the documentation about [steps](/concepts/step) for more information.
 
 ```python
 step = client.api.create_step(
-    thread_id: "1234"
+    thread_id="37822d78-5258-4646-9a68-46906aff5337"
     type = "undefined",
 )
 ```
 
-<ParamField path="thread_id" type="string">
+<ParamField path="thread_id" type="uuid" required>
   The id of the thread
 </ParamField>
 
 <ParamField path="type" type="string">
   The type of the step
 </ParamField>
+<ResponseField name="type" type="string">
```

Comment:
not a type string, type Step

Reply:
and should it have a name?

Reply:
This is a mistake, it should be a ParamField not a ResponseField.
Its fixed

Reply:
The ResponseField acts just like a parameter field : 

![image](https://github.com/Chainlit/platform-docs/assets/8982708/9e7ac485-b5ef-42f9-af66-64d95e63d294)

I need to manually add a `### Return type` before just like in the screenshot. Otherwise it is just mixed like a normal param.

Is this how we want things to look ? I can change it everywhere even if I think it will add more text for not a lot of added value

Reply:
I think it is pretty useful to know the return type of an api